doxygen/CMakeLists.txt

```diff
@@ -20,7 +26,7 @@ set(DOCS_MAINPAGE
 foreach(docFile ${DOCS_MAINPAGE})
```

Comment:
Before the merge though, could somebody quickly explain why we need this again? I'm skimming through the logic and I'm not 100% sure what it's doing

Reply:
Hey @FiniteReality - this is setting the main page to be the contents of `README.md`, and links together the Code of Conduct and the License to the doxygen output (this was added prior to me, but removing it removed all traces of the License, and the main page became blank).

Reply:
Weird. I'll have to check the git blame, I guess :sweat_smile: